### ISSUE TITLE
[Tool bar] `aria-labelledby` pointing to tooltip content is not announced for stories Tooltips and Icon Only

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3275]
### 😯 Current Behavior

Right now user gets no announcement with buttons names that are provided via `aria-labelledby` in stories Tool bar Tooltips and Tool bar Icon Only

https://github.com/user-attachments/assets/40b3b264-715e-4992-98ba-49a228239625

<img width="340" height="268" alt="Image" src="https://github.com/user-attachments/assets/32eb43be-534d-4c0a-80c6-efb12f46103d" />

Partial fix is introduced in PR #3273, but usage of `aria-label` instead of `aria-labelledby` means that whenever you change the tooltip text, you must also remember to update the `aria-label` attribute. This can lead to potential issues in the feature as tooltip would say something different than `aria-label`.

### 🤔 Expected Behavior/possible solution?

Improved version of behavior introduced in PR #3273 should only rely on `aria-labelledby` as it can be observed in [Button Icon Only story](https://storybook.sanomalearning.design/?path=/story/actions-button--icon-only):

https://github.com/user-attachments/assets/d75615ca-090e-480d-8cf3-b7930e7c61de

<img width="341" height="305" alt="Image" src="https://github.com/user-attachments/assets/be34da18-2623-4db2-a6b8-d1e2fea67a63" />

### 🖥️ Steps to Reproduce

1. Go to https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--tooltips or https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--icon-only
2. With screen reader turned on check if names of buttons with tooltips are announced
3. If names are announced check if name is provided with `aria-labelleby`. To do so use browser's Accessibility tab in DevTools.

### Were you able to reproduce this in our storybook?

Yes

### If yes, provide the storybook URL

https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--tooltips 

### ⚖️ Impact

Not a huge problem, we have a workaround

### Framework

Angular

### What browsers are you seeing the problem on?

All

### Component version

v0.2.3

### 👤 Your name

Kacper

### 🧢 Your product/team

SLDS

### 🎨 Your Theme(s)

Sanoma Learning

**Comment by jpzwarte** [comment]
Fixed in #3344
